src/SSL.cpp

```diff
@@ -6,17 +6,32 @@
 #include "murmur_pch.h"
 
 #include "SSL.h"
+#include "SSLLocks.h"
 
 #include "Version.h"
 
 void MumbleSSL::initialize() {
+	// Let Qt initialize its copy of OpenSSL, if it's different than
+	// Mumble's.
```

Comment:
Maybe mention that that is a side effect of calling supportsSsl? While this comment implies it, it could be made explicit.